### PR TITLE
Changed Home page title to explorer by modular cloud

### PR DESCRIPTION
**Pull request by Aswanth-c**
Fixed the mentioned Issue  https://github.com/modularcloud/explorer/issues/62

### CODE REVIEW

**Review by lzrscg — CHANGES_REQUESTED** [review]
This won't work, unfortunately. What needs to happen is we need to edit the whitelabel object (from `getWhitelabel()`) to include a custom whitelabel field for each explorer.

Thanks for getting this started :)

**Comment by lzrscg** [comment]
for example, celestiascan.com should say "CelestiaScan by Modular Cloud"

**Comment by Aswanth-c** [comment]
> This won't work, unfortunately. What needs to happen is we need to edit the whitelabel object (from `getWhitelabel()`) to include a custom whitelabel field for each explorer.
> 
> Thanks for getting this started :)

Hey Lz, sorry for the misunderstanding.

I have fixed the issue by
creating a property  Inside the whiteLableObject variable and naming it "title". 
In most of the cases, I  combined the name array and added those values to the title property. 
except In world and default env where the names  are ["modular" "cloud"], I changed it to explorer

**Comment by lzrscg** [comment]
thanks @Aswanth-c this is great! I was thinking maybe the full title could go in the metadata. So for instance,

"Explorer by Modular Cloud"
"CelestiaScan by Modular Cloud"
"Something else entirely"

Could all be the title.

Perhaps something like this could be added to the whitelabel config:
```
{
   "metadata": {
      "title": "Explorer by Modular Cloud",
      "description": "A modular block explorer built by Modular Cloud",
   }
}
```

**Comment by Aswanth-c** [comment]
> erhaps something like this could be added to the whitelabel config:

Done!

File: apps/web/lib/utils.ts [hunk 1]
```diff
@@ -43,7 +43,10 @@ export async function fetchLoad(props: FetchLoadArgs) {
     return null;
   }
 }
-
+export type metaData = {
```
Comment (lzrscg) [inline apps/web/lib/utils.ts:46]: sorry to be a nitpicker 🥺👉👈 but can you rename this "Metadata"

**Review by lzrscg — CHANGES_REQUESTED** [review]
LGTM after the capitalization fix :D

File: apps/web/lib/utils.ts (hunk 1 above)
Comment (Aswanth-c) [reply apps/web/lib/utils.ts]: Haha nice catch
Fixed!

**Comment by lzrscg** [comment]
i fixed this in a previous update already, thanks for putting this together though!

**Pull request closed without merging**